modules/criteoBidAdapter.js

```diff
@@ -156,18 +156,24 @@ export const spec = {
    * @param {TimedOutBid} timeoutData
    */
   onTimeout: (timeoutData) => {
-    if (publisherTagAvailable()) {
-      // eslint-disable-next-line no-undef
-      const adapter = Criteo.PubTag.Adapters.Prebid.GetAdapter(timeoutData.auctionId);
-      adapter.handleBidTimeout();
+    if (publisherTagAvailable() && timeoutData.length > 0) {
+      var auctionsIds = [];
+      timeoutData.forEach((bid) => {
+        if (!auctionsIds.includes(bid.auctionId)) {
```

Comment:
array `includes` is not supported in IE:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

I recommend using `indexOf`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf

Reply:
Ah yes ! Thanks a lot for catching this